### PR TITLE
Helps 4366 add search aptitude to vs code/npx

### PR DESCRIPTION
**Pull request by aleciberg-olive**

### CODE REVIEW

File: loop-templates/templates/src/aptitudes/index.ts.squirrelly [hunk 1]
```diff
@@ -46,6 +49,9 @@ export {
 {{ @if (it.aptitudes.includes('process')) }}
   processExample,
 {{ /if }}
+{{ @if (it.aptitudes.includes('search')) }}
+  runSearchExample,
```
Comment (ulricesparra-oliveai) [inline loop-templates/templates/src/aptitudes/index.ts.squirrelly:53]: The naming convention here is different than the rest so it seems out of place. Can we change it to `searchExample` and `searchExample.run()`?

File: loop-templates/templates/src/whispers/SearchWhisper.test.ts.squirrelly [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { whisper } from '@oliveai/ldk';
+import SearchWhisper, { onClose } from './SearchWhisper';
+
+jest.mock('@oliveai/ldk');
+
+const mockWhisperClose = jest.fn();
+
+const TEST_PARAM = 'test';
+
+describe('Clipboard Whisper', () => {
```
Comment (ulricesparra-oliveai) [inline loop-templates/templates/src/whispers/SearchWhisper.test.ts.squirrelly:10]:
```suggestion
describe('Search Whisper', () => {
```

**Review by ulricesparra-oliveai — CHANGES_REQUESTED** [review]

**Comment by jonathan-olive** [comment]
It seems like those webpack files Ryan added for his PR are going to be deleted when merged? Might want to double check it and the flowchart as well

**Comment by jonathan-olive** [comment]
Small updates:
`loop-templates/templates/src` needs an update to `index.test.ts.squirrelly`
'vscode-extension` needs a `ChangeLog.md` i'd hope it works nicely w/ git when we want to merge it later

**Pull request merged**